tor/archiver.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import logging
+
+import prawcore
+from praw import Reddit
+
+from tor import config
+from tor.core.initialize import configure_logging
+from tor.core.initialize import configure_tor
+from tor.helpers.misc import explode_gracefully, subreddit_from_url
```

Comment:
That's a valid way of doing it I suppose; I just have a thing against putting multiple imports on one line because when you have a shitload of imports (like some of the projects I have at work) it's difficult to scan the import list and immediately see where an import is.

I'm not terribly preferential to either way of doing it (each on a separate line or what you've got there), just that we decide on it and it goes into the style guide.